### DISCUSSION TITLE
start frigate on docker but, no object detection just movement detection

### DISCUSSION THREAD

**Discussion by sonewtogithub** [blakeblackshear/frigate#8456] [Ask A Question] [answered]
I am new to frigate and want to investigate it before I invest in hardware.
Can I install Frigate in Docker while Home Assistant runs on a virtual machine (both Synology 218+ with 10Gb RAM) 
Can I use Frigate without object regocnition but with movement detection
Answering is fine but tips and tricks are very welcome too

**Comment by NickM-27** [comment]
Motion detection is supported but events are not created, it is just supported for recording only on motion if you want to set it up that way

**Reply by sonewtogithub** [reply]
meaning there is nothing I can use as a trigger for home assistant then?

**Reply by NickM-27** [reply, marked as answer]
oh, yes there is a "motion detection" sensor that can be used in home assistant
